README.md

```diff
@@ -84,6 +84,18 @@ There is a rapidly expanding ecosystem surrounding Reservoir:
 - **[reservoir-docker](https://github.com/mattgrill/reservoir-docker):** Docker image for Reservoir
 - **[well](https://github.com/damontgomery/well):** Drupal installation based on Reservoir and [BLT](https://github.com/acquia/blt)
 
+
+## Drupalists
+
+If you're already deeply familiar with Drupal's concepts (such as entity types, fieldable entity types, config entity types, entity bundles, text formats, vocabularies, Views …) and you are used to finding your way around the Drupal UI, then you may find Reservoir too limiting.
+
+Reservoir limits itself to `node`s ("Content") and `node_type`s ("Content models"). These limitations make it simpler, more understandable, hence simplifying development, maintenance and the mental model.
+
+However, Reservoir still allows you to install any Drupal module to gradually add capabilities (if and when you need them): it's still Drupal under the hood. If you want the most complete Drupal experience, already preconfigured for decoupled use cases, we recommend you try [Contenta](http://www.contentacms.org/).
```

Comment:
@wimleers Actually because we limit some the basic admin routes such as /admin/structure you won't be able to get to configuration that a lot of modules provide